test/e2e/packagemanifest_e2e_test.go

```diff
@@ -178,18 +188,28 @@ func packageManifestHasStatus(pm *packagev1.PackageManifest) bool {
 
 }
 
-func fetchPackageManifest(t GinkgoTInterface, pmc pmversioned.Interface, namespace, name string, check packageManifestCheckFunc) (*packagev1.PackageManifest, error) {
+func fetchPackageManifest(pmc pmversioned.Interface, namespace, name string, check packageManifestCheckFunc) (*packagev1.PackageManifest, error) {
 	var fetched *packagev1.PackageManifest
 	var err error
 
-	err = wait.Poll(pollInterval, pollDuration, func() (bool, error) {
-		t.Logf("Polling...")
+	Eventually(func() (bool, error) {
+		ctx.Ctx().Logf("Polling...")
 		fetched, err = pmc.OperatorsV1().PackageManifests(namespace).Get(context.TODO(), name, metav1.GetOptions{})
 		if err != nil && !errors.IsNotFound(err) {
 			return true, err
 		}
 		return check(fetched), nil
-	})
+	}).Should(BeTrue())
 
 	return fetched, err
 }
+
+func containsPackageManifest(pmList []packagev1.PackageManifest, pkgName string) bool {
```

Comment:
This could be expressed by composing the `ContainsElement` and `WithTransform` matchers. I suspect this is a common pattern in the tests, so maybe that's something to come back to in the future when considering reusable matchers/fixtures.